client/scripts/models/ReactionCount.ts

```diff
@@ -0,0 +1,24 @@
+import { IUniqueId } from './interfaces';
+
+class ReactionCount<T extends IUniqueId> {
```

Comment:
Unused type arg?

Reply:
not sure what do you mean ?

Reply:
You never seem to use the `T` type for anything inside the class, I'm wondering why include it.